packages/core/content-manager/admin/src/preview/index.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable check-file/no-index */
+
+import { FEATURE_ID } from './constants';
+
+import type { PluginDefinition } from '@strapi/admin/strapi-admin';
+
+const previewAdmin = {
+  bootstrap(app) {
+    // TODO: Add license registry check when it's available
+    if (!window.strapi.future.isEnabled(FEATURE_ID)) {
+      return {};
+    }
+    // eslint-disable-next-line no-console -- TODO remove when we have real functionality
+    console.log('Bootstrapping preview admin');
```

Comment:
should this be a dev log instead? this might be confusing for users if we merge into develop

Reply:
what do you mean by dev log? Checking node_env before logging?

Reply:
sorry I meant the `strapi.log.debug` , but I just saw this is under a feature flag so I don't think it matters !